src/msg/async/PosixStack.cc

```diff
@@ -206,6 +206,9 @@ class PosixConnectedSocketImpl final : public ConnectedSocketImpl {
   void close() override {
     ::close(_fd);
   }
+  int set_priority(int sd, int prio, int domain) {
+    handler.set_priority(sd, prio, domain);
```

Comment:
This is missing a return value.

Reply:
i think it's better to change set_priority to void